### PR TITLE
Disable sentry excepthook integration

### PR DESCRIPTION
**Pull request by aapeliv**
It keeps on sending out handled exceptions otherwise

**Backend checklist**
- [ ] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [ ] Run the backend locally and it works
- [ ] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

### CODE REVIEW

File: app/backend/src/app.py [hunk 1]
```diff
@@ -21,7 +27,23 @@
 
 if config.config["SENTRY_ENABLED"]:
     # Sends exception tracebacks to Sentry, a cloud service for collecting exceptions
-    sentry_sdk.init(config.config["SENTRY_URL"], traces_sample_rate=0.0, environment=config.config["COOKIE_DOMAIN"])
+    integrations
```
Comment (kalvdans) [inline app/backend/src/app.py]: syntax error

**Review by kalvdans — COMMENTED** [review]
What is the motivation for not getting the full sentry package?

**Comment by aapeliv** [comment]
> What is the motivation for not getting the full sentry package?

By default it adds an excepthook that will vacuum up unhandled exceptions. I believe it's interfering with gRPC though, and is sending through exceptions that are handled by gRPC. This is polluting the sentry dashboard (and database) with a whole lot of user-errors, e.g. `UNAUTHENTICATED`, `NOT_FOUND`, etc.

File: app/backend/src/app.py (hunk 1 above)
Comment (aapeliv) [reply app/backend/src/app.py]: Thanks for picking that up!!

**Review by kalvdans — APPROVED** [review]
LGTM! Thanks for using trailing commas, it keeps the diff down when adding more parameters.

**Comment by aapeliv** [comment]
> Thanks for using trailing commas, it keeps the diff down when adding more parameters.

It's done by black automatically!

**Pull request merged**